src/wp-content/themes/twentysixteen/functions.php

```diff
@@ -403,23 +403,11 @@ function twentysixteen_scripts() {
 	// Theme block stylesheet.
 	wp_enqueue_style( 'twentysixteen-block-style', get_template_directory_uri() . '/css/blocks.css', array( 'twentysixteen-style' ), '20240817' );
 
-	// Load the Internet Explorer specific stylesheet.
-	wp_enqueue_style( 'twentysixteen-ie', get_template_directory_uri() . '/css/ie.css', array( 'twentysixteen-style' ), '20170530' );
-	wp_style_add_data( 'twentysixteen-ie', 'conditional', 'lt IE 10' );
-
-	// Load the Internet Explorer 8 specific stylesheet.
-	wp_enqueue_style( 'twentysixteen-ie8', get_template_directory_uri() . '/css/ie8.css', array( 'twentysixteen-style' ), '20170530' );
-	wp_style_add_data( 'twentysixteen-ie8', 'conditional', 'lt IE 9' );
-
-	// Load the Internet Explorer 7 specific stylesheet.
-	wp_enqueue_style( 'twentysixteen-ie7', get_template_directory_uri() . '/css/ie7.css', array( 'twentysixteen-style' ), '20170530' );
-	wp_style_add_data( 'twentysixteen-ie7', 'conditional', 'lt IE 8' );
-
-	// Load the html5 shiv.
-	wp_enqueue_script( 'twentysixteen-html5', get_template_directory_uri() . '/js/html5.js', array(), '3.7.3' );
-	wp_script_add_data( 'twentysixteen-html5', 'conditional', 'lt IE 9' );
-
-	// Skip-link fix is no longer enqueued by default.
+	// Register handles for removed stylesheets and scripts.
+	wp_register_style( 'twentysixteen-ie', false, array( 'twentysixteen-style' ) );
+	wp_register_style( 'twentysixteen-ie8', false, array( 'twentysixteen-style' ) );
+	wp_register_style( 'twentysixteen-ie7', false, array( 'twentysixteen-style' ) );
+	wp_register_script( 'twentysixteen-html5', false, array() );
```

Comment:
The empty array is default for dependencies.

```suggestion
	wp_register_script( 'twentysixteen-html5', false );
```